tests/python/test_torch_io.py

```diff
@@ -156,7 +156,7 @@ def test_io_simple():
 
 @ti.torch_test
 def test_io_simple():
```

Comment:
Also, why here 2 two `test_io_simple`, they share the same name...

Reply:
It's very likely a typo..